src/engine/Match_search_mode.ml

```diff
@@ -617,6 +619,14 @@ let rec filter_ranges (env : env) (xs : (RM.t * MV.bindings list) list)
               *)
              | Some capture_bindings -> Some (r, capture_bindings @ new_bindings)
              )
+         | R.CondAnalysis (mvar, CondEntropyV2) -> begin
+              match !hook_pro_entropy_analysis with
+              | None -> failwith "Core entropy plugin not installed"
```

Comment:
opted for failwith - open to suggestions for how this should behave. Rule should explode spectacularly and not run, but probably shouldn't hose the whole run if there's an unsupported rule in a suite.